### ISSUE TITLE
Highlight dates in between range on mouse over

### ISSUE THREAD

**Issue by multiwebinc** [nazar-pc/PickMeUp#208]
When using a range, after the first date is selected, the values in between should be highlighted when mousing over the dates on the calendar. Similar to https://flatpickr.js.org/examples/#range-calendar.

**Comment by multiwebinc** [comment]
@nazar-pc This can't be done with pure CSS. I'm thinking you misunderstood what I was talking about. I mean highlight the dates in between the two dates. Sort of like it is now after you select the second date, however doing that on mouse over **before** the second date is actually selected.

**Comment by nazar-pc** [comment]
No, I heard you right. You can add event handler for mouse movements and add classes accordingly yourself. Sure, not as simple as having it built-in, but still  very straightforward to do.
